### DISCUSSION TITLE
Welcome to dtn7-rs Discussions!

### DISCUSSION THREAD

**Discussion by gh0st42** [dtn7/dtn7-rs#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blueberrylxm** [comment]
Hello respected developer! I have configured DTN7 rs on my device, but I am not sure how to enable it now. Can you provide me with some help?

**Reply by gh0st42** [reply]
You can start the bundle protocol agent by running `dtnd` on the command line.
The configuration can be either provided as command line arguments or through a configuration file.
Some inspiration can be found in the [getting started guide](https://github.com/dtn7/dtn7-rs/blob/master/doc/getting-started.md) or through the use of different parameters in the [tests](https://github.com/dtn7/dtn7-rs/tree/master/tests).

**Comment by febrian-hub** [comment]
Hellooo anyone help me... 

i'm so sorry to ask this twice as if spamming because i have made discussion before. 
@gh0st42  could you visit this discussion page? thank you!

https://github.com/dtn7/dtn7-rs/discussions/66
